interface1.js

```diff
@@ -421,14 +421,22 @@ window.addEventListener('load', function(){
 
 		//Get input to GEN.
 		var pString = spotForm.inputToGen.value;
-
 		// Get the code that is in the stree textarea
 		var treeCode = spotForm.sTree.value
 		// if code has been generated, then ignore pString in GEN
 		if(treeCode !== "{}") {
 			pString = "";
 		}
-
+		if(document.getElementById('inputOptions').style.display == 'block') {
+			if (spotForm.inputToGen.value != ""){
```

Comment:
We also need to check if there is content in the code version of the list of trees.

Reply:
I just implemented this since it was a quick fix.